github4s/shared/src/test/scala/github4s/utils/TestData.scala

```diff
@@ -74,6 +74,7 @@ trait TestData extends DummyGithubUrls {
   val nonExistentSearchQuery = "nonExistentSearchQueryString"
   val validSearchParams = List(
     OwnerParamInRepository(s"$validRepoOwner/$validRepoName"),
+    LabelParam(label = "bug"),
```

Comment:
Unfortunately this will fail since this test is used to retrieve the issue #40 which is labelled enhancement, that's what I meant in my review sorry if I wasn't clear.

Reply:
Oh hah!